atlp-rwanda/atlp-pulse-fn
# FIX(bug): A bug that causes the profile image to stay static.

# PR Description
It fixes the design bugs in the admin dashboard.
# Description of tasks that were expected to be completed

- [x] It is fixing the profile image of the trainee so that it won't keep hiding the details of the trainee.
- [x] It is changing the close icon for the trainee list from the "x" letter to the universal close symbol.


# How has this been tested?

- Log in as the admin, then access the trainees' ratings and profiles to observe changes in both profile picture behavior and the design of the close button.
- Log in as the admin, then navigate to the team's model within the "Teams and Cohorts" sections in the left sidebar.

# Number of Commits

The number of commits should not exceed **2 commits**. In case they are more than that, please take your time and squash them.

# Screenshots (If appropriate)

![msedge_7zfR3kMYTD](https://github.com/atlp-rwanda/atlp-pulse-fn/assets/87605095/ba19983b-b29d-4ff2-873f-42681799393d)
![msedge_pjabCbYsYW](https://github.com/atlp-rwanda/atlp-pulse-fn/assets/87605095/e3025184-1502-4c07-9caa-6aa4585e3d69)
![msedge_uPuJKdMxjl](https://github.com/atlp-rwanda/atlp-pulse-fn/assets/87605095/52bcee0e-c621-4445-bbe2-a7dd0c098c68)

# Please check this Checklist before you submit your PR:
